colossalai/fx/_compatibility.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable
+
+import torch
+
+from . import META_COMPATIBILITY
+
+
+def compatibility(is_backward_compatible: bool = False) -> Callable:
+    """A decorator to make a function compatible with different versions of PyTorch.
+
+    Args:
+        is_backward_compatible (bool, optional): Whether the function is backward compatible. Defaults to False.
+
+    Returns:
+        Callable: The decorated function
+    """
+
+    def decorator(func):
+        if META_COMPATIBILITY:
+            return func
+        else:
+            if is_backward_compatible:
+                return func
+            else:
+
+                def wrapper(*args, **kwargs):
+                    raise RuntimeError(f'Function `{func.__name__}` is not compatible with PyTorch {torch.__version__}')
+
+                return wrapper
+
+    return decorator
```

Comment:
This is a decorator that wraps our defined function. If the PyTorch version is not satisfied, we can wrap the function to be a `RuntimeError`.

Reply:
What is the difference between this and the PyTorch one?